src/unity/python/turicreate/data_structures/sframe.py

```diff
@@ -3824,6 +3824,16 @@ def groupby(self, key_column_names, operations, *args):
         --------
         aggregate
 
+        Notes
+        -----
+        * Numeric aggregator (such as sum, mean, stdev etc.) follow the skip
+        None policy i.e they will omit all missing values from the aggregation.
+        As an example, `sum([None, 5, 10]) = 15` because the `None` values is
+        skipped.
+        * Aggregators have a default value when no values (after skipping all
```

Comment:
What are the default values for the other aggregators? I think it would be useful to list them comprehensively here.

Reply:
Fixed.